### PR TITLE
Expose run microtasks

### PR DESCRIPTION
**Pull request by benjamingr**
Fixes https://github.com/sinonjs/lolex/issues/147 

cc @SimenB @fatso83

### CODE REVIEW

**Comment by SimenB** [comment]
Nice!

File: src/lolex-src.js [hunk 1]
```diff
@@ -557,7 +557,11 @@ function createClock(start, loopLimit) {
 
     function updateHrTime(newNow) {
         clock.hrNow += (newNow - clock.now);
-    }
+    };
```
Comment (SimenB) [inline src/lolex-src.js]: CI is unhappy about this semi

Comment (benjamingr) [reply src/lolex-src.js]: CI is right, it makes no sense to end a function statement with a semicolon :D

**Comment by fatso83** [comment]
The build failed on 2 of 3 Node versions. Restarting now to see if it was transient (the log only said `null`)

**Comment by fatso83** [comment]
Thank you! The build errors were just transient on Travis.

**Pull request merged**